### PR TITLE
Mint Signup Form Clone Project

### PR DESCRIPTION
**Pull request by nusretbutunay**
This project consists of building an HTML document that matches the appearance of mint.com’s signup page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+          <label for="email">Email Address</label>
+          <input type="email" name="email" id="email" />
+          <label for="phone">Phone (recommended)</label>
+          <input type="tel" name="phone" id="phone" />
+          <p>Standard call, messaging or data rates may apply.</p>
+          <label for="password">Password</label>
+          <input type="password" name="password" id="password" />
+          <button class="signin create-btn">
+            <i class="fa fa-lock" aria-hidden="true"></i>Create Account
+          </button>
```
Comment (HADDADSOHAIB) [inline index.html]: The button should have a type='submit':
- [x] It has a `<submit>` button that matches the style of the original page (Blue background, white text) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Sohaib Haddad. (TSE)
### Change Required :recycle: 

### Hello @nusretbutunay and @KartheekWD     :handshake: 

### You did wonderful job :clap:  your project is looking good :medal_sports: and look like the original.

### There is just a small issue ( I am sure you will fix it in no time), check the comments for more details.

### Once the changes are implemented, send another code review request.

**Comment by nusretbutunay** [comment]
Create Account Button type fixed

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi, @nusretbutunay and @KartheekWD, great job :+1: Congratulations on the awesome work :slightly_smiling_face: You fixed the comment by the previous TSE and now the project has met all the minimum requirements. I hereby approve your project :smile: 

Go ahead and merge to master, do not forget to submit the project completion form
See you at the top :top: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**